src/test/java/uk/ac/ebi/eva/evaseqcol/service/SeqColServiceTest.java

```diff
@@ -90,4 +92,24 @@ void unbalancedDuplicatesPresentTest() {
         assertFalse(seqColService.unbalancedDuplicatesPresent(A2, B2));
         assertTrue(seqColService.unbalancedDuplicatesPresent(A3, B3));
     }
+
+    @Test
+    void A_And_B_Same_OrderTest() {
+        List<String> listA1 = new ArrayList<>(Arrays.asList("chr1", "chr2", "chr3", "M"));
+        List<String> listA2 = new ArrayList<>(Arrays.asList("1", "2", "2", "3"));
+        List<String> listA3 = new ArrayList<>(Arrays.asList("ch1", "B", "ch2", "ch3"));
+        List<String> listA4 = new ArrayList<>(Arrays.asList("ch1", "B", "ch2", "ch3"));
+
+        List<String> listB1 = new ArrayList<>(Arrays.asList("chr1", "chr2", "chr3"));
+        List<String> listB2 = new ArrayList<>(Arrays.asList("1", "2", "2"));
+        List<String> listB3 = new ArrayList<>(Arrays.asList("ch1", "A", "ch2", "ch3"));
+        List<String> listB4 = new ArrayList<>(Arrays.asList("A", "ch1", "ch2", "ch3"));
+
+        assertTrue(seqColService.check_A_And_B_Same_Order(listA1, listB1));
+        assertTrue(seqColService.check_A_And_B_Same_Order(listA2, listB2));
+        assertTrue(seqColService.check_A_And_B_Same_Order(listA3, listB3));
+        assertTrue(seqColService.check_A_And_B_Same_Order(listA4, listB4));
+
```

Comment:
Might want to add a negative example, e.g. `A = [1,2,2]` and `B = [2,1,2]` should be false.